File: mine/src/main/java/com/klcw/app/mine/dataload/MinDataLoad.java
package com.klcw.app.mine.dataload;

import com.billy.android.preloader.interfaces.GroupedDataLoader;

/**
 * @Auther: yd
 * @datetime: 2018/10/24
 * @desc:
 */
public class MinDataLoad implements GroupedDataLoader<String> {

    public static final String MINE_LIST_KEY = "MinDataLoadKey";

    public MinDataLoad(String par) {

    }

    @Override
    public String keyInGroup() {
        return MINE_LIST_KEY;
    }

    @Override
    public String loadData() {
        return "你好";
    }
}

File: mine/src/main/java/com/kk/app/mine/dataload/MinDataLoad.kt
package com.kk.app.mine.dataload

import com.billy.android.preloader.interfaces.GroupedDataLoader

/**
 * @author kk
 * @datetime: 2018/10/24
 * @desc:
 */
class MinDataLoad(par: String?) : GroupedDataLoader<String> {
    override fun keyInGroup(): String {
        return MINE_LIST_KEY
    }

    override fun loadData(): String {
        return "你好"
    }

    companion object {
        const val MINE_LIST_KEY = "MinDataLoadKey"
    }
}